apps/namehashlabs.org/components/2 - molecules/products-section.tsx

```diff
@@ -98,7 +97,7 @@ const Product = ({
                     target={withoutExternalLinkIconInCTA ? undefined : "_blank"}
```

Comment:
Not sure if I'm leaving this comment in the right place, but can you please investigate the UX of this button:

![CleanShot 2024-09-21 at 00 21 17](https://github.com/user-attachments/assets/854ff84b-add0-4a26-9a05-96674f6ce0db)

A few issues here:
1. I'm in the Vercel preview environment, but for some reason this link is going to the production website.
2. This link is internal and therefore should be a `NextLink`
3. This link shouldn't open in a new tab.

Reply:
Fixed.